libr/bin/p/bin_pe.c

```diff
@@ -108,9 +108,8 @@ static char *signature (RBinFile *bf, bool json) {
 	}
 	struct PE_ (r_bin_pe_obj_t) * bin = bf->o->bin_obj;
 	if (json) {
-		RJSVar *json = r_pkcs7_cms_json (bin->cms);
-		char *c = r_json_stringify (json, false);
-		r_json_var_free (json);
+		PJ *pj = r_pkcs7_cms_json (bin->cms);
+		char *c = pj_string (pj);
```

Comment:
does PJ require to be freed before returning?

Reply:
i've checked the usage of it.
you forgot to free it.

Reply:
If I free it there I got:

```
r2(69885,0x1001295c0) malloc: *** error for object 0x103024000: pointer being freed was not allocated
r2(69885,0x1001295c0) malloc: *** set a breakpoint in malloc_error_break to debug
```

Reply:
Where should I free it?

Reply:
```c
PJ *pj = r_pkcs7_cms_json (bin->cms);
char *c = pj_string (pj);
pj_free (pj);
return c;
```